### ISSUE TITLE
Select Tests instead of All Org Tests being run

### ISSUE THREAD

**Issue by datapharmer** [afawcett/githubsfdeploy#33]
There should be an option to run only tests from github or only selected tests instead of running all tests in the org.

**Comment by afawcett** [comment]
Interesting... so its running all unmanaged tests in the org and not just the repo tests? If so, that should be any easy fix as i think there is now a header option to run only the deployed tests.

**Comment by datapharmer** [comment]
Correct, that was my experience. Adding the header option would certainly be useful.

**Comment by jnylund** [comment]
Can you folks give me more info about the header option and how it is used, I am trying to deploy this package to prod - https://github.com/jongpie/ApexUuid and it is running all the unit tests (others which break) are failing the deployment.

**Comment by afawcett** [comment]
@jnylund this is a code change to the tool that is needed, its not something you can effect, sorry.

**Comment by Garywoo** [comment]
I would also find this useful. We have some managed packages in our org whose tests are incompatible with having Country/State Picklists enabled. Since we have this enabled in our org, and it's not something you want to quickly toggle off for deployment, selecting tests to run, so we can avoid those ones running during deployment with this tool would be very useful.
